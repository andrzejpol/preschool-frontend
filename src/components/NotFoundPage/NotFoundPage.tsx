const NotFoundPage = () => {

  return (
    <div
      className="w-screen h-screen flex flex-col items-center justify-center bg-gradient-to-b from-preschoolPrimary to-preschoolPrimaryLight">
      <h1 className="text-6xl font-bold text-gray-800">404</h1>
      <p className="text-2xl mt-4 text-gray-600">Oops! The page you're looking for doesn't exist.</p>
    </div>
  )
};

export default NotFoundPage;