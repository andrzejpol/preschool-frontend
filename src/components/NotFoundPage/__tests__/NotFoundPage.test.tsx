import React from "react";
import {render} from "@testing-library/react";
import NotFoundPage from "../NotFoundPage";

describe("NotFoundPage Component", () => {
  it("matches the snapshot", () => {
    const {asFragment} = render(<NotFoundPage/>);

    // Snapshot assertion
    expect(asFragment()).toMatchSnapshot();
  });
});
