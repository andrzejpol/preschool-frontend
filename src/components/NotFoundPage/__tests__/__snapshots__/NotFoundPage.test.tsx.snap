// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFoundPage Component > matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="w-screen h-screen flex flex-col items-center justify-center bg-gradient-to-b from-preschoolPrimary to-preschoolPrimaryLight"
  >
    <h1
      class="text-6xl font-bold text-gray-800"
    >
      404
    </h1>
    <p
      class="text-2xl mt-4 text-gray-600"
    >
      Oops! The page you're looking for doesn't exist.
    </p>
  </div>
</DocumentFragment>
`;
